run.zsh

```diff
@@ -102,9 +120,10 @@ get_avg_startup() {
   local startup_time startup_total startup_avg
   local i n
 
-  startup_times=($(cut ${results_dir}/${1}.log -c49-53))
+  startup_times=($(sed -e 's/.*cpu //' -e 's/ total//' ${results_dir}/${1}.log))
   for i in ${startup_times}; do (( startup_total += ${i} )); done
-  for n in ${#startup_times}; do (( startup_avg = ${startup_total} / ${n} )); done
+  (( startup_avg = ${startup_total} / ${#startup_times} * 1000 ))
+  startup_avg=$(printf "%.0f ms" ${startup_avg})
```

Comment:
When I wrote [zsh-startup-timer](https://github.com/paulmelnikow/zsh-startup-timer) I found this format to be the most legible. Don't think these are really accurate beyond millisecond resolution (or that we care).